libs/@hashintel/design-system/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@hashintel/design-system",
-  "version": "0.0.6",
+  "version": "0.0.7",
```

Comment:
Changesets will apply any increase you specify in the changeset file, you don't need to amend version numbers

Reply:
Reverted in 0ea36e14639ab60d6f34272101c72b6a58a29520